### PR TITLE
[0112] fix broken link

### PR DESCRIPTION
**Pull request by katesalazar**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Is there any value in linking to a BIP 9 draft, when there is already a link to the actual BIP 9?

**Comment by katesalazar** [comment]
I'm not interested in proposing a change to this document,
but I will applaud it.

**Comment by kallewoof** [comment]
For historical posterity? Doesn't seem very important, but we should address broken links. Perhaps that link should simply be removed.

File: bip-0112.mediawiki [hunk 1]
```diff
@@ -388,7 +388,7 @@ Thanks to Eric Lombrozo and Anthony Towns for contributing example use cases.
 
 [http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-August/010396.html Softfork deployment considerations]
 
-[https://gist.github.com/sipa/bf69659f43e763540550 Version bits]
+[https://web.archive.org/web/20210925124425/https://gist.github.com/sipa/bf69659f43e763540550 Version bits]
```
Comment (devnullada) [inline bip-0112.mediawiki:391]:
```suggestion

```
```suggestion
[https://web.archive.org/web/20210925124425/https://gist.github.com/sipa/bf69659f43e763540550 Version bits]
```

Comment (devnullada) [inline bip-0112.mediawiki:391]:
```suggestion

```
```suggestion
[https://web.archive.org/web/20210925124425/https://gist.github.com/sipa/bf69659f43e763540550 Version bits]
```

**Comment by luke-jr** [comment]
@btcdrak @maaku @CodeShark

**Comment by katesalazar** [comment]
> [luke-jr](https://github.com/luke-jr) added the [Proposed BIP modification](https://github.com/bitcoin/bips/labels/Proposed%20BIP%20modification) label [5 hours ago](https://github.com/bitcoin/bips/pull/1276#event-6557260940)

I was trying to avoid being labeled Proposed BIP modification...

**Comment by luke-jr** [comment]
That makes no sense.

**Comment by maaku** [comment]
Approve.

Sent from my mobile device;
please excuse typos and brevity.

**Pull request merged**